pkg/internal/sqlprune/xwbparameterized_test.go

```diff
@@ -0,0 +1,1081 @@
+/*
+Copyright 2017 Google Inc.
```

Comment:
Why do we need these tests in our repo pkg folder? I think they should be in vendor? DId you modify the  xwb tests?

Reply:
We're using xwb's Scan() to lex, but we have our own simplistic parser that extracts the operation and table name based on the type of tokens that were found.  I have two sets of this: the original (w/ changed package name,) and the parameterized version where many of the ID tokens have been changed to question mark placeholders.  I expect our parser is going to see parameterized queries as the majority of cases.  Given that it does provide useful exercise to the sqlprune parser, it's only present in _test.go files, I'm not sure of the downside of leaving this stuff here - what do you think, should it still be removed?

Reply:
Ah I see, thanks for explaining, keep them in then, no special downside except test time, but these are fast anyway.